test/e2e/scenario/internalMonitoring.scenario.ts

```diff
@@ -15,6 +15,7 @@ describe('internal monitoring', () => {
             throw new window.Error('bar')
           },
         }
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
The ESLint rule is wrong in this case, it says that the `!` not needed. But it's needed because `window.DD_LOGS` may be `undefined`. This behavior may be associated with this open issue typescript-eslint/typescript-eslint#2248